api/apps/v1alpha1/nimservice_types.go

```diff
@@ -85,6 +91,26 @@ type NIMServiceSpec struct {
 	GroupID          *int64     `json:"groupID,omitempty"`
 	RuntimeClassName string     `json:"runtimeClassName,omitempty"`
 	Proxy            *ProxySpec `json:"proxy,omitempty"`
+	// MultiNode config for NIMService using LeaderWorkerSet
+	MultiNode *NimServiceMultiNodeConfig `json:"multiNode,omitempty"`
+}
+
+// NimServiceMultiNodeConfig defines the configuration for multi-node NIMService.
+// TODO disable Scale when this is enabled
+type NimServiceMultiNodeConfig struct {
+	// +kubebuilder:validation:Minimum=1
+	// +kubebuilder:default:=2
```

Comment:
I thought we will not limit it to minimum 2 workers to allow testing on single node correct?